src/Sentry/Platforms/Android/Extensions/SentryEventExtensions.cs

```diff
@@ -31,6 +37,11 @@ public static SentryEvent ToSentryEvent(this JavaSdk.SentryEvent sentryEvent, Ja
 
     public static JavaSdk.SentryEvent ToJavaSentryEvent(this SentryEvent sentryEvent, SentryOptions options, JavaSdk.SentryOptions javaOptions)
     {
+        if (sentryEvent.Sdk != null)
+        {
+            sentryEvent.Sdk.Name ??= "Sentry";
+            sentryEvent.Sdk.Version ??= "1.0.0";
```

Comment:
Here again, I think we can set defaults as [per the Enricher](https://github.com/getsentry/sentry-dotnet/blob/ee62bb942b6bebc6b835130a5192aa68a31b6f54/src/Sentry/Internal/Enricher.cs#L61-L68)